package.json

```diff
@@ -46,7 +47,6 @@
         "@types/sinon": "~17.0.0",
         "@types/yargs": "~17.0.22",
         "chai": "^4.3.7",
-        "cross-env": "^7.0.3",
```

Comment:
we no longer need `cross-env` with Yarn v4; this is now built-in to the script runner